UnityProject/Assets/Scripts/Core/Input System/InteractionV2/TileInteraction/PipeDeconstruction.cs

```diff
@@ -18,11 +17,24 @@ public class PipeDeconstruction : TileInteraction
 
 		[Tooltip("Action message to performer when they begin this interaction.")]
 		[SerializeField]
-		private string performerStartActionMessage = " you unwrench the pipe ";
+		private string performerStartActionMessage = null;
```

Comment:
This would clear the message on anything that relied on the default message. Is that intentional?

Reply:
the wrench pipe deconstruction interaction should be the only thing using this which ive changed anyway so it shouldn't matter